renovate.json5

```
@@ -0,0 +1,11 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "local>hivemq/renovate-config:default.json5"
+    ],
+    baseBranches: [
+        "master",
+        "renovate-playground"
+    ],
+    "useBaseBranchConfig": "merge"
```

```suggestion
    "useBaseBranchConfig": "merge",
    "branchPrefix": "renovate/hivemq-community-edition/"
```
Let's add the branchPrefix for consistency.